ansible/roles/irods-icommands/meta/main.yml

```diff
@@ -0,0 +1,197 @@
+galaxy_info:
+  author: Chris Martin, Andre Mercer
+  description: Installs iRODS iCommands 4.1.9, including irodsFs
+  company: CyVerse
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: BSD
+
+  min_ansible_version: 2.1
+
+  # Optionally specify the branch Galaxy will use when accessing the GitHub
+  # repo for this role. During role install, if no tags are available,
+  # Galaxy will use this branch. During import Galaxy will access files on
+  # this branch. If travis integration is cofigured, only notification for this
+  # branch will be accepted. Otherwise, in all cases, the repo's default branch
+  # (usually master) will be used.
+  #github_branch:
+
+  #
+  # Below are all platforms currently available. Just uncomment
+  # the ones that apply to your role. If you don't see your
+  # platform on this list, let us know and we'll get it added!
+  #
+  platforms:
+  - name: EL
+    versions:
+    - all
+    - 5
+    - 6
+    - 7
+  #- name: GenericUNIX
```

Comment:
@steve-gregory we don't need them, it's just 'scaffolding' carried over from [ansible-role-template](https://github.com/cyverse-ansible/ansible-role-template) via `ansible-galaxy init`. If you don't want want it in the codebase then I can rip it out of [ansible-irods-icommands](https://github.com/CyVerse-Ansible/ansible-irods-icommands) and re-install the role here, updating this PR.